README.md

```diff
@@ -247,7 +247,8 @@ easy to treat every value as a list, and should discourage treating values
 passed as `Value.List`s specially.
 
 API users should be encouraged to return unbracketed comma-separated lists
-unless there's a particular reason not to.
+unless there's a particular reason not to. Only empty and single-element lists
+can have an undecided separator.
```

Comment:
I think this should probably go in the documentation for `List.separator`, since that's usually where we put field-by-field requirements.